sklearn/svm/classes.py

```diff
@@ -594,7 +594,7 @@ def __init__(self, nu=0.5, kernel='rbf', degree=3, gamma=0.0,
 
 
 class SVR(BaseLibSVM, RegressorMixin):
-    """epsilon-Support Vector Regression.
+    """Epsilon-Support Vector Regression.
```

Comment:
But then the Nu should be also small in the other algorithms, shouldn't it? I could also change the nu. Whatever is your preference, but for consistency both should be the same.